cbioportal_etl/scripts/diff_studies.py

```diff
@@ -496,7 +516,7 @@ def run_py(args):
             update_only_ids=update_only_ids,
             current_attr_only=current_only_attr,
             update_attr_only=update_only_attr,
-            attr_cts=update_delta_attr_cts,
+            attr_cts=delta_attr_cts,
```

Comment:
`delta_attr_cts` not used...may is supposed to be `delta_attr_count` ?

Reply:
Correct